app/lib/efile/line_data.yml

```diff
@@ -652,6 +652,8 @@ NJ1040_LINE_51:
   label: '51 Use Tax Due on Internet, Mail-Order, or Other Out-of-State Purchases (See instructions) If no Use Tax, enter 0.00'
 NJ1040_LINE_56:
   label: '56 Property Tax Credit (See instructions page 25)'
+NJ1040_LINE_57:
+  label: '57 New Jersey Estimated Tax Payments/Credit from 2022 tax return'
```

Comment:
is there a need to make `2022` dynamic or worded as "prior year"?

Reply:
A good question... I'm honestly not sure how this line_data is used

Reply:
hallo, line data is used in the "Explain Calculations". 
<img width="622" alt="Screenshot 2024-11-07 at 9 14 48 AM" src="https://github.com/user-attachments/assets/832d356a-01d8-460d-91d8-1b4765266d7a">

Line data is the not-bolded bit under the Line ID. Basically used internally by programs/product folks to explain what went into the calcuations/what the value is representing
<img width="906" alt="Screenshot 2024-11-07 at 9 14 54 AM" src="https://github.com/user-attachments/assets/fb16efc7-cc13-47ed-9d11-f9030c5382e9">

Reply:
oh that is cool to know! Since it's internal i wouldn't invest too much in making dynamic, though generic "prior year" wording might still be a good idea so it never needs an update